config/manifests/bases/cockroach-operator.clusterserviceversion.yaml

```diff
@@ -23,46 +23,94 @@ metadata:
     createdAt: CREATED_AT_PLACEHOLDER
     description: CockroachDB Operator
     operators.operatorframework.io/builder: operator-sdk-v1.1.0
-    operators.operatorframework.io/project_layout: go.kubebuilder.io/v2
+    operators.operatorframework.io/project_layout: go.kubebuilder.io/v3
```

Comment:
None of the changes in this file are necessary for this PR. This is what _will_ be generated when we do a release. I left it here so people could see what the changes will be for the upcoming release